.github/workflows/create_release_assets.yml

```diff
@@ -51,6 +56,22 @@ jobs:
         if: ${{ matrix.platform != 'windows-latest' }}
         shell: bash
 
+      - name: Build Debian-based system binary and create package
+        run: |
+          rm -rf target/release
+          cargo build --release --features default
+          cargo deb --no-build --no-strip
+        if: ${{ matrix.platform == 'ubuntu-latest' }}
+        shell: bash
+
+      - name: Move Debian-based system package
+        run: |
+          mkdir -p assets
+          mv target/debian/*.deb assets
+          ls assets
```

Comment:
Seems for debugging purposes only, can be removed if so.

Reply:
Yeah, that's there because I saw an `ls .` at the end of `Rename Release (Unix)` and I wanted to match the behavior.  I can remove it.

Reply:
I didn't notice that, sorry<